test/DynamoCoreWpfTests/ViewExtensions/DocumentationBrowserViewExtensionTests.cs

```diff
@@ -470,7 +470,8 @@ public void RemovesScriptTagsFromLoadedHtml()
                 Assert.AreEqual(0, tabsBeforeExternalEventTrigger);
                 Assert.AreEqual(1, tabsAfterExternalEventTrigger);
                 Assert.IsTrue(htmlContent.Contains("<h2 id=\"heading\">Division by zero</h2>"));
-                Assert.False(htmlContent.Contains("document.getElementById(\"heading\").innerHTML = \"Script1\";"));
+                var message = string.Format("hml: {0}", htmlContent);
+                Assert.False(htmlContent.Contains("document.getElementById(\"heading\").innerHTML = \"Script1\";"),message);
```

Comment:
@QilongTang @reddyashish 
After this change the `DocumentationBrowserViewExtensionTests.RemovesScriptTagsFromLoadedHtml() `test passed (not sure why, probably is a timing issue), it is always passing for me locally using VS2022 and using **_dotnet test_**.
I will trigger again two times the job for executing the tests and check that continue passing.